quickstart-guide.md

```diff
@@ -9,19 +9,20 @@ description: The quickest way to get your first workflow deployed on Aqueduct
 First things first, we'll install the Aqueduct pip package and start the Aqueduct server and UI:
 
 ```bash
-pip3 install aqueduct-ml
-aqueduct server & # The & here will start this as a background process.
-aqueduct ui &
+pip3 install aqueduct-ml torch transformers
+aqueduct start & # The & here will start this as a background process.
```

Comment:
Let's drop the ampersand here and below -- there's some funny reason that I haven't been able to figure out that makes you need to hit enter before you get your terminal window back.